src/components/users/viewCohorts/components/Table.js

```diff
@@ -0,0 +1,162 @@
+        >
+            <Table aria-label='simple table'>
+                <TableHead
+                    sx={{
+                        backgroundColor: '#23232c',
+                    }}
+                >
+                    <TableRow>
+                        <TableCell
+                            sx={{
+                                color: '#ffffff',
+                                fontWeight: 'bold',
+                                borderBlockColor: '#23232c',
+                                textAlign: 'center',
+                            }}
+                        >
+                            Student ID
+                        </TableCell>
+                        <TableCell
+                            sx={{
+                                color: '#ffffff',
+                                fontWeight: 'bold',
+                                borderBlockColor: '#23232c',
+                                textAlign: 'center',
+                            }}
+                        >
+                            Profile
+                        </TableCell>
+                        <TableCell
+                            sx={{
+                                color: '#ffffff',
+                                fontWeight: 'bold',
+                                borderBlockColor: '#23232c',
+                                textAlign: 'center',
+                            }}
+                        >
+                            FirstName
+                        </TableCell>
+                        <TableCell
+                            sx={{
+                                color: '#ffffff',
+                                fontWeight: 'bold',
+                                borderBlockColor: '#23232c',
+                                textAlign: 'center',
+                            }}
+                        >
+                            LastName
+                        </TableCell>
+                        <TableCell
+                            sx={{
+                                color: '#ffffff',
+                                fontWeight: 'bold',
+                                borderBlockColor: '#23232c',
+                                textAlign: 'center',
+                            }}
+                        >
+                            Email
+                        </TableCell>
+                        <TableCell
+                            sx={{
+                                color: '#ffffff',
+                                fontWeight: 'bold',
+                                borderBlockColor: '#23232c',
+                                textAlign: 'center',
+                            }}
+                        >
+                            GitHub
+                        </TableCell>
+                    </TableRow>
+                </TableHead>
+                <TableBody>
+                    {cohortStudents.map((row, key) => (
```

Comment:
Similarly to the other comment, I think this component can be split into smaller components.

Reply:
Sorted 👍